Category-Colors/category-colors
# Feature/scrollable containers

# Description
- The left sidebar and main body should scroll independently.

Issue # 
- issue #35 (let me know if i need to update this)

Describe what you're adding
- Added scrollable feature to both sidebar and main-card containers

# QA

List out how to test your changes and add a screenshot of what it should look like, when applicable.
- you can test changes by checking out the 'feature/scrollable-containers' branch
- see screenshot attached 
- ![scrollableContainers](https://github.com/Category-Colors/category-colors/assets/57161327/1c637923-5119-42cb-9e65-9698b84a7836)
